src/components/cards/NewFeatureCard.vue

```diff
@@ -1,19 +1,36 @@
 <template>
-  <timed-card startTime="March 6, 2025" endTime="March 8, 2025">
+  <timed-card startTime="May 8, 2025" endTime="May 26, 2025">
     <div class="container">
       <div class="title"></div>
       <div class="message">
-        <a class="link" href="/snowblitz" target="_blank">❄️ Operation Snowball's Snowblitz is TODAY!</a> Don't miss out on a night of fun activities, team building, bonding with friends, and pizza! 
+        SHS's 2024-25 Music Production 2 classes engaged in an activist songwriting project centered around important topics to raise awareness for
+        <a class="link" href="https://www.gkmcenglewood.com/" target="_blank">Good Kids Mad City</a>,
```

Comment:
Based on Joey's comment, let's axe the specific callouts to these orgs and make the language a bit more neutral (i.e., "for important causes"). It's better to stay neutral here.